documents/src/index.md

```diff
@@ -1,24 +1,18 @@
 <!--
 type: page
-title: Element Framework
+title: Element Framework (V6)
 description: Welcome to Element Framework
 location: ./
 layout: default
 -->
 
+<div style="color:black;width:100%;background-color:#FFE999;padding:15px 20px 30px 20px;">This is the documentation for <b>Element Framework v6</b>. We have recently released a new shiny version. You can access it from <a target="_blank" href="https://ui.refinitiv.com">https://ui.refinitiv.com</a>.</div>
```

Comment:
we probably need to use `http://ui.refinitiv.com` instead of `https://ui.refinitiv.com` to workaround client navigation.

Reply:
Updated example snippet